src/View/View.php

```diff
@@ -702,6 +702,13 @@ public function render(?string $template = null, $layout = null): string
         $this->dispatchEvent('View.afterRender', [$templateFileName]);
 
         if ($this->autoLayout) {
+            if (empty($this->layout)) {
+                throw new RuntimeException(
+                    'View::$layout must be a non-empty string.' .
+                    'To disable layout rendering set View::$autoLayout to false instead.'
```

Comment:
Should we tell folks to use the disableAutoLayout() method instead of modifying the property?

Reply:
Done